qemu/tests/cfg/nic_acpi_index.cfg

```diff
@@ -0,0 +1,28 @@
+- nic_acpi_index:
+    virt_test_type = qemu
+    type = nic_acpi_index
+    only Linux
+    no q35
```

Comment:
After checking https://wiki.qemu.org/ChangeLog/6.0#ACPI, seems that acpi-index only supports pc platform currently, so you can update the restriction accordingly.

Reply:
Hi @PaulYuuu You mean: I should use `only pc` to restrict instead of `no q35`, is it right?

Reply:
`only i440fx`

Reply:
Hi @PaulYuuu Updated, please help me review it again, thanks a lot.